src/theme.js

```diff
@@ -1,3 +1,7 @@
-const { default: CreateMuiTheme } = require('@material-ui/core/styles/createMuiTheme');
+const theme = {
+    colors: {
+        duckBill: '#DB2F2A'
```

Comment:
I may swap this for a more generic name like "primary" or "secondary," similar to how we did this with the material-ui theme in [the material branch](https://github.com/BigRoomStudios/strangeluv/pull/237#discussion_r336609394)